test/common/grpc/common_test.cc

```diff
@@ -71,12 +71,76 @@ TEST(GrpcCommonTest, ChargeStats) {
 }
 
 TEST(GrpcCommonTest, PrepareHeaders) {
-  Http::MessagePtr message = Common::prepareHeaders("cluster", "service_name", "method_name");
+  {
+    Http::MessagePtr message = Common::prepareHeaders("cluster", "service_name", "method_name",
+                                                      absl::optional<std::chrono::milliseconds>());
```

Comment:
absl::nullopt (throughout)

Reply:
Done, did not find further instances of this.